beckn/mobility
# adding domain specific rules/restrictions in the mobility on demand layer2 config

### **Description**

Networks created using Beckn Onix use two layers of configurations for the request and response validation in a transaction.  

As part of this PR, an example layer 2 configuration for the mobility on demand rides use case is created. 

### **Changes**

1. Added domain rules in the OpenAPI document "api/l2-config/mobility_ondemandride_1.1.0.yaml".
2. A postman collection, to be created to test the network rules added in the layer 2 config.

### **Tools needed to test this PR**

Postman Desktop or Web Application.

### **How to test**

1. Import the postman collection inside the "api/l2-config/postman-collection-test-cases/ folder into the postman application. 
3. The postman collection contains requests, these requests are to be named in a way that clearly conveys their intended purpose.
4. Send the requests to verify that the added rule are working as expected.

### **Notes**

This is an example configuration for a Layer 2 network. Network facilitators setting up a new network may use this as a reference. Adjustments may be necessary to suit specific network requirements and conditions.
